### PR TITLE
Fix 218

### PR DESCRIPTION
**Pull request by microsarah**

### CODE REVIEW

File: 05_chapter/06_exercise.md [hunk 1]
```diff
@@ -17,48 +17,50 @@ Here's the editor you should use to write your code.
...
 1 . Set your div's font-size to 1.2em, and tell it to only stretch across 50% of the screen's width.
 
-2 . Tell both divs to display as inline-block elements.
+2 . Add the about ID to the first div and the services ID to the second div.
 
-3 . Create CSS rules (selectors) for #about and #services.
+3 . Tell both divs to display as inline-block elements.
 
-4 . Give #about a float of left.
+4 . Create CSS rules (selectors) for #about and #services.
```
Comment (jshawl) [inline 05_chapter/06_exercise.md]: should use backticks -  `#about`

File: 05_chapter/06_exercise.md [hunk 2]
```diff
@@ -17,48 +17,50 @@ Here's the editor you should use to write your code.
...
 
-5 . Give #services a float of right.
+5 . Give #about a float of left.
```
Comment (jshawl) [inline 05_chapter/06_exercise.md]: backticks here

File: 05_chapter/06_exercise.md [hunk 3]
```diff
@@ -17,48 +17,50 @@ Here's the editor you should use to write your code.
...
 
 At this point, you'll notice that some things are not so awesome here.
   * The two columns are smushed together.
   * The footer is behaving oddly.
 
-6 . Let's start with the quickest fix. Give the footer a clear value of both.
+7 . Let's start with the quickest fix. Give the footer a clear value of both.
 
-7 . The footer's in the right place, but the two columns are still too close! What do you know about adding whitespace between HTML elements? Currently, #services and #about are 50% of the page width each. Here's some math. You're borrowing from the page's width to add a 2% padding on all four sides, so that each div keeps its total width of 50%. So go ahead and a width of 46% and padding of 2%.
+8 . The footer's in the right place, but the two columns are still too close! What do you know about adding whitespace between HTML elements? Currently, #services and #about are 50% of the page width each. Here's some math. You're borrowing from the page's width to add a 2% padding on all four sides, so that each div keeps its total width of 50%. So go ahead and add a width of 46% and padding of 2%.
```
Comment (jshawl) [inline 05_chapter/06_exercise.md]: add backticks for ids here

Comment (jshawl) [inline 05_chapter/06_exercise.md]: also the 46% ans 2% is an anti-pattern. instead, should be `box-sizing:border-box`

File: 05_chapter/06_exercise.md [hunk 4]
```diff
@@ -17,48 +17,50 @@ Here's the editor you should use to write your code.
...
-12 . Flip to your HTML, and wrap a new div, called "hero", around the main image.
+13 . Flip to your HTML, and wrap a new div, called "hero", around the main image.
 
 *Your img might freak out. Don't worry. It's because the "hero" div is grabbing the default div style written for your columns. Let's prevent that from happening. You need to counteract the div's width, margin, and display values.*
 
-13 . In your HTML, wrap a section around both divs: open it after the `</header>` closing tag, and close it before the opening `<footer>` tag.
+14 . In your HTML, wrap a section around both divs: open it after the `</header>` closing tag, and close it before the opening `<footer>` tag.
 
-14 . Create a CSS style for the section. Set its background to #222222, and make its color white. Make the section's overflow:auto.
+15 . Create a CSS style for the section. Set its background to #222222, and make its color white. Make the section's overflow:auto.
```
Comment (jshawl) [inline 05_chapter/06_exercise.md]: make its color `#ffffff`

File: 05_chapter/06_exercise.md (hunk 3 above)
Comment (microsarah) [inline 05_chapter/06_exercise.md]: have they learned box-sizing yet tho?

Comment (jshawl) [inline 05_chapter/06_exercise.md]: should be added here, but no https://github.com/GA-WDI/fundamentals/blame/d0aa553f132eef465ff936a50756f4d352b9b204/04_chapter/03_lesson.md#L12

**Pull request merged**